recipes/config.rb

```diff
@@ -49,6 +49,16 @@
   mode '00755'
 end
 
+if node['cassandra']['tmp_dir']
+  directory "#{node['cassandra']['tmp_dir']}" do
+    action :create
+    recursive true
+    owner node['cassandra']['user']
+    group node['cassandra']['group']
+    mode '0755'
```

Comment:
Using:
`only_if node['cassandra']['tmp_dir']`

Instead of the if statement is a little neater